### PR TITLE
fix: update to latest alphas, screenshots, and rebased from main

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes: https://github.com/patternfly/patternfly-org/issues/3924
Closes: https://github.com/patternfly/patternfly-org/issues/3923

- updates to core alpha: 6.0.0-alpha.103
- updates to react alphas: 6.0.0-alpha.43
- updates screenshots (includes updates to use pf-v6-u-h-100)
- merges changes made to main into v6 (mostly a11y docs and design guideline updates)

Fixes: https://github.com/patternfly/patternfly-org/issues/3928
Fixes: https://github.com/patternfly/patternfly-org/issues/3927

### CODE REVIEW

File: packages/documentation-framework/versions.json [hunk 1]
```diff
@@ -20,8 +20,81 @@
         "@patternfly/quickstarts": "5.1.0",
         "@patternfly/react-virtualized-extension": "5.1.0"
```
Comment (kmcfaul) [inline packages/documentation-framework/versions.json:21]: Can we add the react-templates package here too?

File: packages/documentation-site/patternfly-docs/content/get-started/get-started.css [hunk 2]
```diff
@@ -2,6 +2,11 @@
   color: #0066CC;
 }
 
+.ws-images {
+  margin: auto;
+  width: 600p;
```
Comment (mcoker) [inline packages/documentation-site/patternfly-docs/content/get-started/get-started.css:7]:
```suggestion
  width: 600px;
```

**Review by mcoker — COMMENTED** [review]
one small typo

**Comment by nicolethoen** [comment]
I am happy to address follow up issues in subsequent PRs - The changes to the docs-framework here will clean up a lot for the core/react workspaces and give the staging site a lot of badly needed polish. 👍🏻

**Pull request merged**